### PR TITLE
[GSW-31] fix: Token Details Improve UI

### PR DESCRIPTION
**Pull request by khiemldk**

### CODE REVIEW

File: packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx [hunk 1]
```diff
@@ -1,56 +1,196 @@
...
+    setFromData(toData);
+    setToData(fromData);
+  };
+
+  const handleCopied = async () => {
+    try {
+      await navigator.clipboard.writeText(window.location.href);
+      setCopied(true);
+      setTimeout(() => {
+        setCopied(false);
```
Comment (harryoh) [inline packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx:147]: What does calling setCopied every 2 seconds mean?

**Comment by harryoh** [comment]
Could you fix unittest after working your task?
`yarn test:ci` on packages/web
or `yarn workspace @gnoswap-labs/web test:ci` on root

File: packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx (hunk 1 above)
Comment (khiemldk) [reply packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx:147]: Just want to dispay tooltip 2 seconds @harryoh

**Comment by harryoh** [comment]
And one more thing.
Add Jira key to commit message, please.
like `[GSW-31] fix: Fix bug vercel token detail`

File: packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx (hunk 1 above)
Comment (harryoh) [reply packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx:147]: I think calling it every 2 seconds for a tooltip is too inefficient.
Is there any other way?

Comment (khiemldk) [reply packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx:147]: It just show tooltip one time after click @harryoh

Comment (harryoh) [reply packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx:147]: my mistake. I mis-read it as `setInterval`.
no problems. :)

**Comment by khiemldk** [comment]
> And one more thing. Add Jira key to commit message, please. like `[GSW-31] fix: Fix bug vercel token detail`

It's okay now

**Comment by khiemldk** [comment]
> Could you fix unittest after working your task? `yarn test:ci` on packages/web or `yarn workspace @gnoswap-labs/web test:ci` on root

It's okay now @harryoh

**Review by harryoh — APPROVED** [review]
👍

**Pull request merged**